tests/pausing_test.go

```diff
@@ -422,13 +422,12 @@ var _ = Describe("[rfe_id:3064][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			tests.WaitForVMICondition(virtClient, vmi, v1.VirtualMachineInstancePaused, 30)
 
 			By("Waiting longer than the process normally runs")
-			time.Sleep(10 * time.Second)
+			time.Sleep(70 * time.Second)
```

Comment:
You should use a constant (here and in the arg to `sleep`, to make sure the next person that touches this does not break it again.

But frankly, I don't like this change as it wastes 60s for no good reason. There are better ways to test that time inside the guest does not progress as fast as in the host when the guest is paused.

Reply:
> You should use a constant (here and in the arg to `sleep`, to make sure the next person that touches this does not break it again.
> 
Done.
> But frankly, I don't like this change as it wastes 60s for no good reason. There are better ways to test that time inside the guest does not progress as fast as in the host when the guest is paused.

Are we testing the time the vmi is successfully paused here, or that running processes are continued after the vmi is paused?
I understood the second option and acted upon that, if it is the first option then maybe the whole tests case needs rethinking.

Reply:
The important bit is the former (showing that pause really pauses the guest). The second is implicit. The fact that you *can* use the same console and continue sending commands to bash proves it. You can be more explicit if you want by verifying that your shell process id (`$$`) did not change after pause.

Reply:
> The important bit is the former (showing that pause really pauses the guest). The second is implicit. The fact that you _can_ use the same console and continue sending commands to bash proves it. 

Thanks you for explaining.
> You can be more explicit if you want by verifying that your shell process id (`$$`) did not change after pause.

Awesome! Done.

Reply:
OK. Thank you for explaining.
I changed the test , to not use the long process and instead use the uptime diff.